src/components/NegotiationList.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
Bootstrap isn't needed here, since it is already imported in main.js. Bootstrap 5 is used